macro/src/main/scala/monocle/macros/GenIso.scala

```diff
@@ -59,21 +60,22 @@ private object GenIsoImpl extends MacrosCompatibility {
     """)
   }
 
-  def genIso_unit_impl[S: c.WeakTypeTag](c: Context): c.Expr[Iso[S, Unit]] = {
+  def genIso_unit_impl[S: c.WeakTypeTag]: c.Expr[Iso[S, Unit]] = {
     import c.universe._
 
     val sTpe = weakTypeOf[S]
 
     if (sTpe.typeSymbol.isModuleClass) {
       val table = c.universe.asInstanceOf[SymbolTable]
-      val obj = makeAttributedQualifier(c)(table.gen, sTpe)
```

Comment:
Would be worth adding `Context#internal.gen.mkAttributedQualifier` to macro-compat too.